### DISCUSSION TITLE
How to contribute to the SDK?

### DISCUSSION THREAD

**Discussion by qwertychouskie** [FIRST-Tech-Challenge/FtcRobotController#817] [General]
https://github.com/FIRST-Tech-Challenge/FtcRobotController/blob/master/.github/CONTRIBUTING.md says this:

> Pull requests that do not fall into the category above are evaluated by the FTC Technology Team on a case-by-case basis. Please note however that the deployment model of the SDK does not support direct pulls into ftctechnh/ftc_app.

However, if one does try to open a PR (e.g. https://github.com/FIRST-Tech-Challenge/FtcRobotController/pull/802), it gets auto-closed with this message:

> Thank you for your interest in the FTC SDK. Most pull requests landing here are from team forks and are not intended to be merged. For the rest, due to the nature of the FTC SDK release process, FIRST does not accept pull requests on the FtcRobotController repository. If you would like a change, please open an issue.

However, issues or issue comments that have the code changes often do not receive a response (e.g. https://github.com/FIRST-Tech-Challenge/FtcRobotController/issues/383#issuecomment-1587693828).

So as someone who wants to help make the SDK better, what is the proper method for contributing?

P.S. There was a similar discussion topic that was posted a while back (https://github.com/FIRST-Tech-Challenge/FtcRobotController/discussions/93) but it sadly never got a response to the core question of how to contribute.

**Comment by AlecHub** [comment]
If you browse through [Issues](../issues) you will find many areas that FIRST Tech Challenge should be proactive about but isn't even willing to be reactive about.  This is something that the new Senior Director would be interested in looking into (according to her).  She can be reached at https://reddit.com/u/-Rach-
